lib/mastodon/accounts_cli.rb

```diff
@@ -397,19 +397,25 @@ def refresh(username = nil)
         end
 
         say("Refreshed #{processed} accounts#{dry_run}", :green, true)
-      elsif username.present?
-        username, domain = username.split('@')
-        account = Account.find_remote(username, domain)
-
-        if account.nil?
-          say('No such account', :red)
-          exit(1)
-        end
+      elsif usernames.size > 0
+        usernames.each do |user|
+          user, domain = user.split('@')
+          account = Account.find_remote(user, domain)
+
+          if account.nil?
+            say('No such account', :red)
+            exit(1)
+          end
 
-        unless options[:dry_run]
-          account.reset_avatar!
-          account.reset_header!
-          account.save
+          unless options[:dry_run]
+            begin
+              account.reset_avatar!
+              account.reset_header!
+              account.save
+            rescue Mastodon::UnexpectedResponseError
+              say('Account failed ' + user + "@" + domain, :red)
+            end
+          end
```

Comment:
The linter has a couple good suggestions here:
```suggestion
          next if options[:dry_run]

          begin
            account.reset_avatar!
            account.reset_header!
            account.save
          rescue Mastodon::UnexpectedResponseError
            say("Account failed: #{user}@#{domain}", :red)
          end
```